packages/volto-slate/src/blocks/Text/index.js

```diff
@@ -118,6 +121,27 @@ export default function applyConfig(config) {
       addPermission: [],
       view: [],
     },
+    cloneData: (data) => {
+      return [
+        uuid(),
+        {
+          ...data,
+          value: [
+            ...(data?.value || []).map((c) => {
+              return {
+                ...c,
+                children: (c?.children || []).map((childrenData) => {
+                  return {
+                    ...childrenData,
+                    data: { ...childrenData.data, uid: nanoid(5) },
```

Comment:
This will add a uid even if there wasn't one there before; is that what we want?

Reply:
I think that all children have uid by default

Reply:
No, I don't think so. I have not seen it before.